src/completion-provider/template-completion-provider.ts

```diff
@@ -72,7 +72,7 @@ export default class TemplateCompletionProvider {
 function listComponents(root: string): CompletionItem[] {
   const jsPaths = walkSync(join(root, 'app', 'components'), {
     directories: false,
-    globs: ['**/*.js']
+    globs: ['**/*.js', '**/*.hbs']
```

Comment:
I know it is unrelated to this specific PR, but seems like we should find `*.ts` as well no?

Reply:
Yeah, sounds like a good improvement.

Reply:
Tracking at https://github.com/ember-tooling/ember-language-server/issues/358